src/app/not-found.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import Link from "next/link";
+import { Plugs } from "phosphor-react";
+
+import { Button } from "@/components/Button";
+
+export default function NotFound() {
+  return (
+    <div className="absolute flex flex-col min-h-screen items-center justify-center p-5 w-full bg-white z-10">
+      <div className="lg:px-24 lg:py-24 md:py-20 md:px-44 px-4 py-24 items-center flex justify-center flex-col-reverse lg:flex-row md:gap-28 gap-16">
+        <div className="xl:pt-24 w-full xl:w-1/2 relative pb-12 lg:pb-0">
+          <div className="relative">
+            <div className="absolute">
+              <div className="">
+                <h1 className="my-2 text-gray-800 font-bold text-2xl">
+                  Página não encontrada!
+                </h1>
+                <p className="my-2 text-gray-800">
+                  Desculpe por isso! Volte para a página principal!
+                </p>
+                <Button className="sm:w-full lg:w-auto my-2 border rounded md py-4 px-8 text-center bg-green-600 text-white hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-700 focus:ring-opacity-50">
+                  <Link href="/">Voltar</Link>
+                </Button>
```

Comment:
É melhor evitar ter um link dentro de um botão, e, além disso, o botão não possui um atributo type especificado nem uma ação definida para quando for clicado. Se a razão for puramente estética, considere aplicar os estilos diretamente ao link e remover o botão. Se a necessidade for de um botão, você pode redirecionar para a página desejada através da função onclick do botão.